enerdata/contracts/normalized_power.py

```diff
@@ -87,3 +87,7 @@ def get_volt_int(self, pot):
 
     def is_normalized(self, pot):
         return pot in NORMALIZED_POWERS
+
+    def get_norm_powers(self, pot_min, pot_max):
+        return [norm_pow for norm_pow in sorted(NORMALIZED_POWERS)
+                if pot_min < norm_pow <= pot_max]
```

Comment:
I think in this case maybe is not very important for performance but it is an example for possible future situations. Imagine that `NORMALIZED_POWERS` has 1000 items and you only want the some powers that are in the beginning of the list.
With this algorithm you will loop for **all** the powers checking `if pot_min < norm_pow <= pot_max`. Maybe if you already have sorted the list you can put a `break` when you reach a `pot_norm` that is greater than `pot_max`